server_test.go

```diff
@@ -61,6 +62,10 @@ var (
 	routeWithJSON = rocket.Get("route_with_json/:field", func(r *RouteWithJSON) string {
 		return r.Field + r.Query + r.JField
 	})
+	mime = rocket.Get("/mime/*filename", func(fs *Files) *response.Response {
+		return response.File(fs.FileName).
+			SetContentType(response.ByFileNameSuffix())
+	})
```

Comment:
The way to use `response.File` is little confused me.
I think `response.File` should automatically set `Content-Type`.
If user don't want to use original Content-Type, it can rewrite the Content-Type by this way.
```go
response.File()
  .Headers(
    response.Headers{
      "Content-Type": "the content-type user want",
    },
 )
```

Reply:
Then we don’t need File actually, the only reason that make it exists is let response a file easier, and set header content-type is not the way

Reply:
We still need the `response.File` to help the user parse file and set Content-Type.